internal: fix build errror

transforming (76) ..\web\src\components\xtermWrapper.csspackages/web/src/shared/prompts.ts:17:45 - error TS2307: Cannot find module '@testIsomorphic/types' or its corresponding type declarations.

17 import type { MetadataWithCommitInfo } from '@testIsomorphic/types';
                                               ~~~~~~~~~~~~~~~~~~~~~~~

[6:53:23 PM] Found 1 error. Watching for file changes.

--- a/packages/recorder/tsconfig.json
+++ b/packages/recorder/tsconfig.json
@@ -20,6 +20,7 @@
       "@isomorphic/*": ["../playwright-core/src/utils/isomorphic/*"],
       "@protocol/*": ["../protocol/src/*"],
       "@recorder/*": ["../recorder/src/*"],
+      "@testIsomorphic/*": ["../playwright/src/isomorphic/*"],
       "@web/*": ["../web/src/*"],
     }
   },

--- a/packages/recorder/tsconfig.node.json
+++ b/packages/recorder/tsconfig.node.json
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "composite": true,
     "module": "esnext",
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true
   },
   "include": ["vite.config.ts"]
 }

